### PR TITLE
CI: Enable Travis with submodule check

### PR DESCRIPTION
**Pull request by nonifier**
Add Travis yaml that checkouts Navitia and call a script that makes sure submodules are all merged in !

### CODE REVIEW

**Review by TeXitoi — APPROVED** [review]

**Comment by TeXitoi** [comment]
@nonifier please repush to trigger travis, I've enable it on CanalTP/navitia

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+language: cpp
+
+git:
+  submodules: false
+
+before_install:
+  - sed -i 's,git\@github.com:\([^/]*\)/\(.*\).git,https://github.com/\1/\2,' .gitmodules
+  - git submodule update --init --recursive
+
+install:
```
Comment (kinnou02) [inline .travis.yml]: It should be `script` I think

**Comment by TeXitoi** [comment]
@nonifier you can just `git commit --amend && git push -f`

travis runned and failed!

**Comment by nonifier** [comment]
It works, it has failed :)

**Review by xlqian — APPROVED** [review]

**Review by kinnou02 — APPROVED** [review]

**Pull request merged**